README.md

```diff
@@ -30,45 +32,40 @@ Name your file with your GitHub username `username.json`; for example mine was `
   "location": ""
 }
 ```
-
-Notes on the information:
-
-> - **Name**: You can use your github name, actual name, or a pseudonym!
-
-> - **Url**: Please insert Github link here, we will expand the user json files to include socials in the future
-
-> - **Image**: You can use any _appropriate_ profile image here
-
-> - **Why** : Please write down your reason why you have joined the #FitDevs community!
-
-> - **Weekly**: Your goals for fitness, can be food/exercise/mental/emotional, whatever fitness means to YOU
-
-```
-`weekly` examples :
-`["Running 6x", "Abs 6x", "Pushups 6x"]`
-`["Drinking 500ml Water 3/day", "Meatless Mondays 2x/month", "Sleeping 7h 6/week"]`
+<br>
+and fill out the information (make sure to look at the notes below)<br><br>
+
+ - Name: You can use your github name, actual name, or a pseudonym!
+ - Url: Please insert Github link here, we will expand the user json files to include socials in the future
+ - Image: You can use any appropriate profile image here
+ - Why: Please write down your reason why you have joined the #FitDevs community!
+ - Weekly: Your goals for fitness, can be food/exercise/mental/emotional, whatever fitness means to YOU
+ > Such as the below
+```json
+["Running 6x", "Abs 6x", "Pushups 6x"]
+["Drinking 500ml Water 3/day", "Meatless Mondays 2x/month", "Sleeping 7h 6/week"]
 ```
-
-> - **Goal**: What is your overarching goal? We will discuss this in an upcoming twitter space!
-
-> - **Location**: Be as vague or as specific as you would like to
-
-Example: ![image](https://user-images.githubusercontent.com/96984912/180859482-a59aa1a7-2df6-4900-9ad5-201667205c50.png)
-
-3. Commit the new file!
-
-![image](https://user-images.githubusercontent.com/109629064/180862145-2d9e9d55-4099-4660-81fd-fad3a60372f1.png)
-
-Select Create a new Branch!
-
-![image](https://user-images.githubusercontent.com/109629064/180862218-99a9942e-5911-4534-9c69-180a87e99bd2.png)
-
-4. Create a pull request!
-
-Select the green button: "Create a pull request"
-
-![image](https://user-images.githubusercontent.com/109629064/180862468-449829fb-e71f-4964-bc6b-c547f15920ea.png)
-
-Select "Create Pull Request" on the next page
-
-![image](https://user-images.githubusercontent.com/109629064/180862550-20279183-d4e2-4367-a709-bfac1305cf67.png)
+ - Goal: What is your overarching goal?
+ - Location: Be as vague or as specific as you would like to
```

Comment:
I think we should bold the fields similar to how they were before. It reads a little more clear to me